### DISCUSSION TITLE
2024-03-11-r1

### DISCUSSION THREAD

**Discussion by Zoey2936** [ZoeyVid/NPMplus#697] [Announcements]
## What's Changed
* Mention healthcheck when migrating in README by @YeapGuy in https://github.com/ZoeyVid/NPMplus/pull/692
* fix and close #696
* improve chmod/chown commands in start.sh
* small dep updates

## New Contributors
* @YeapGuy made their first contribution in https://github.com/ZoeyVid/NPMplus/pull/692

**Full Changelog**: https://github.com/ZoeyVid/NPMplus/compare/2024-03-09-r1...2024-03-11-r1

<hr /><em>This discussion was created from the release <a href='https://github.com/ZoeyVid/NPMplus/releases/tag/2024-03-11-r1'>2024-03-11-r1</a>.</em>

**Comment by scampower3** [comment]
Fails to start when i updated to latest. Works when I revert back to 389.

**Reply by Zoey2936** [reply]
can you send the logs?

**Reply by scampower3** [reply]
https://pastebin.com/WnX1pAu6

**Reply by scampower3** [reply]
in the newer image nginx.pid and error.log files are missing

**Reply by Zoey2936** [reply]
will be fixed in the next release
